src/Template/Element/Form/core_properties.twig

```diff
@@ -1,18 +1,45 @@
 {% set attributes = data.attributes %}
+{% set module = currentModule.name|default('') %}
+{% set core_properties = config('Modules.' ~ module ~ '.core_attributes')|default(['title', 'description', 'uname', 'status']) %}
 
 <div class="tab"><h2>{{ __('Main Properties') }}</h2></div>
 
 <fieldset id="core_properties">
     {{ Form.hidden('id', {'value': object.id})|raw }}
-    {{ Form.control('title')|raw }}
-    {{ Form.control('description', {'type': 'textarea'})|raw }}
-    {{ Form.control('uname')|raw }}
-    {{ Form.control('status', {
-        'type': 'radio',
-        'options': [
-            {'value': 'on', 'text': __('On')},
-            {'value': 'draft', 'text': __('Draft')},
-            {'value': 'off', 'text': __('Off')},
-        ]
-    })|raw }}
+
+
+{% for key, property in core_properties %}
+    {% if property == 'password' %}
+    <div class="password-component">
+        {{ Form.control('password', {
+            'class': 'password',
+            'placeholder': 'new password',
+            'autocomplete': 'new-password',
+            'default': ''
+        })|raw }}
+        {{ Form.control('password', {
```

Comment:
It is not a good idea to have two `Form.control('password', {})`. This should be something else, like `Form.control('password-confirm', {})`, and `name` should not be passed as an option to this method.

Reply:
I see. It was done like this in another issue, I just moved the snippet from a twig to another one. BTW I'll amend this part as you suggest ;)

Reply:
Yeah, I see you just moved it into another place… but still I don't like that. 😄